app/services/find_programme_activities.rb

```diff
@@ -1,16 +1,17 @@
 class FindProgrammeActivities
   include Pundit::Authorization
 
-  attr_accessor :organisation, :user, :fund_code
+  attr_accessor :organisation, :user, :fund_code, :is_oda
 
-  def initialize(organisation:, user:, fund_code: nil)
+  def initialize(organisation:, user:, fund_code: nil, is_oda: [nil, true, false])
```

Comment:
Could we make `is_oda` a single boolean here and do this inside `#call`? I found it a little difficult to parse when reading where this is initialized in `programme_activities`

Reply:
Maybe a verbose `include_ispf_non_oda_activities` that defaults to `false` to keep it in sync with the rest of the way the class has been structured?